cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -65,6 +66,8 @@ public class CQL2PgJSON {
   private static Logger logger = LogManager.getLogger(CQL2PgJSON.class);
 
   private static final String JSONB_COLUMN_NAME = "jsonb";
+  /** map schemaPath to Schema */
+  private static Map<String, Schema> dbSchemaCache = new HashMap<>();
```

Comment:
We should consider using a `ConcurrentHashMap` here. At the minimum, multiple eventloops may try to modify contents of the schema. Also this class does not communicate that it should be used in a vertx only context. 
Most of the modifications will happen at the beginning of the app so I don't think it will be a big issue.